### PR TITLE
Bring vtebench up to date

### PR DESCRIPTION
**Pull request by kchibisov**
This is a series of commits updating various parts of vtebench.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,15 +1,16 @@
 [package]
 name = "vtebench"
-version = "0.1.2"
+version = "0.2.0"
+authors = ["Christian Duerr <contact@christianduerr.com>", "Joe Wilm <joe@jwilm.com>"]
 license = "Apache-2.0"
-authors = ["Joe Wilm <joe@jwilm.com>"]
 description = "Benchmark terminal emulators"
-documentation = "https://docs.rs/vtebench"
```
Comment (chrisduerr) [inline Cargo.toml:7]: Why was this removed?

File: Cargo.toml [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
-structopt-derive = "0.2.10"
-structopt = "0.2.10"
-terminfo = "0.6.1"
-failure = "0.1.2"
-rand = "0.5.5"
+structopt-derive = "0.4"
+structopt = "0.3"
+terminfo = "0.7"
+failure = "0.1"
+rand = "0.7"
```
Comment (chrisduerr) [inline Cargo.toml]: Can we go ahead and add the patch version to all of these?

File: src/main.rs [hunk 3]
```diff
@@ -1,26 +1,9 @@
...
 //! A tool for generating benchmark scripts for terminal emulators
 //!
 //! This program is intended to be run and its output piped into a file, and the
 //! file can simply be `cat`ed from the terminal emulator under test. This
 //! ensures that the terminal is being benchmarked and not this vtebench
 //! application.
-extern crate rand;
-extern crate structopt;
-
 #[macro_use]
```
Comment (chrisduerr) [inline src/main.rs]: We should probably remove all the `#[macro_use]` annotations in favor of direct imports of the macros. To comply with the 2018 edition stuff.

**Review by chrisduerr — CHANGES_REQUESTED** [review]
Looks mostly good, thanks. Just to clarify these commits should be rebased and not squashed.

File: Cargo.toml (hunk 1 above)
Comment (kchibisov) [reply Cargo.toml:7]: It's a binary and not a lib, there's no value in such thing.

File: src/main.rs (hunk 3 above)
Comment (kchibisov) [reply src/main.rs]: That's not a bad idea, I'll try to do so.

File: Cargo.toml (hunk 2 above)
Comment (kchibisov) [reply Cargo.toml]: I'm not entirely sure that we should, since there's no real reason for it, and users may have a version behind on their systems, though,

File: Cargo.toml (hunk 1 above)
Comment (chrisduerr) [reply Cargo.toml:7]: Oh, fair point. I forgot about that.

File: Cargo.toml (hunk 2 above)
Comment (chrisduerr) [reply Cargo.toml]: Well did you test it with the 0.X.0 versions? If not, we definitely should. Otherwise we should test with the versions set to 0.X.0 and then specify those exactly, to make it clear that's what works. There's rarely ever a good reason to omit the patch version.

File: Cargo.toml [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
-version = "0.1.2"
+version = "0.2.0"
+authors = ["Christian Duerr <contact@christianduerr.com>", "Joe Wilm <joe@jwilm.com>"]
 license = "Apache-2.0"
-authors = ["Joe Wilm <joe@jwilm.com>"]
 description = "Benchmark terminal emulators"
-documentation = "https://docs.rs/vtebench"
-repository = "https://github.com/alacritty/vtebench"
+readme = "README.md"
+homepage = "https://github.com/alacritty/vtebench"
```
Comment (chrisduerr) [inline Cargo.toml]:
```suggestion
repository = "https://github.com/alacritty/vtebench"
```

**Review by chrisduerr — CHANGES_REQUESTED** [review]
Except for the small suggestion I've left, the overall changes seem fine. But looking at the individual commits seems a bit strange. Did something get messed up during rebase?

Seems like some changes slipped into unrelated commits.

**Comment by kchibisov** [comment]
>Seems like some changes slipped into unrelated commits.

It seems like that, but it was actually intentionally, but I'm not sure why I've done it that way.

**Comment by chrisduerr** [comment]
> It seems like that, but it was actually intentionally, but I'm not sure why I've done it that way.

Seems like it doesn't make sense the way it is right now? So probably best to change it.

**Comment by kchibisov** [comment]
I've updated commits, should be fine now.

**Review by chrisduerr — APPROVED** [review]
LGTM
